lib/listening.js

```diff
@@ -1,5 +1,5 @@
 // Packages
-const {copy} = require('copy-paste')
+const {write: copy} = require('clipboardy')
```

Comment:
`clipboardy.write` returns a promise already so the `promisify` call below is unnecessary.